### DISCUSSION TITLE
BT and ET sensors not showing data

### DISCUSSION THREAD

**Discussion by joeb32** [artisan-roaster-scope/artisan#724] [General]
Hi there, I'm looking for some help on a current issue I am having.

I have just got a new computer (MacBook Air M1) previously had old 2012 Macbook, however when I go to run a roast the BT and ET sensors are not recording anything. However on the Phidget control panel they are showing the temperatures. I am using the exactly the same artisan settings as I was on my old computer. please see images below. 

Any ideas as to how I can fix this issue?

Thanks

<img width="1021" alt="Screenshot 2021-11-14 at 16 29 22" src="https://user-images.githubusercontent.com/94323521/141701609-c8f939f4-0268-4439-8491-e9c3bc084f09.png">

<img width="1182" alt="Screenshot 2021-11-14 at 16 29 32" src="https://user-images.githubusercontent.com/94323521/141701615-9cd874d8-ef7f-46b9-bb9d-ddababcf8d68.png">
![IMG_2279](https://user-images.githubusercontent.com/94323521/141701695-f526c5f9-7762-4cd0-904d-6c5f8d4246d7.jpeg)

![IMG_2280](https://user-images.githubusercontent.com/94323521/141701698-a11ed592-ec24-4ecf-8caa-ffdb9610fc35.jpeg)

**Comment by MAKOMO** [comment]
Did you install the Phidget "Standalone Control Panel" on the top of this page: "https://www.phidgets.com/docs/OS_-_macOS" or the regular driver a bit further down under "OS X 10.11+: Installer Download"?

**Reply by joeb32** [reply]
I’m pretty sure I installed the “OS X 10.11+: installer download” further down the page. 

Shall I download and install “standalone control panel”?

thanks for your help

**Comment by MAKOMO** [comment]
No, the standalone control panel does not install the driver system wide as far as I understand.

I do not see a specific reason why Artisan should not be able to access the Phidget driver while the Phidget Control Panel (if it is not that standalone one) can. Only reason could be some access restrictions by the operating system. Sorry, but we do not have access to a M1 system right now to test and investigate.

Only thing you could try is to install a continuous build from

https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

as this one contains some Phidget logging. To activate this after installation, hold the OPTION and COMMAND keys and click the plus icon. Now try to connect Artisan to your Phidgets. After this failed (just guessing), hold the OPTION key and click the plus icon again. This should open an email with the Artisan log file as well as the Phidget log file attached. You might attach both files here per drag-and-drop into a message box on Github and we will take a look.

**Reply by joeb32** [reply]
I also tried on a windows laptop and I had the same issue of Artisan not reading the sensors.

I just clicked the link you sent but it only looks like there is a windows file to download not a Mac file. is there a Mac file I can download? I could send my saved Artisan settings file if that is of any help?

Thanks

**Reply by MAKOMO** [reply]
Driver installed on Windows? Very strange.

Could you check again for the macOS dmg file. They disappear while new builds are produced. A build takes some minutes up to half an hour. Now the builds are back online.

**Comment by CarefreeBuzzBuzz** [comment]
Did you approve Phidgets in the Security Panel.  Pretty sure you have to.  I just set up a new Mac and it was required.  
----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Reply by joeb32** [reply]
I haven't no! How do I do this, I cant see anything for it on 'security and privacy'?

Thanks

**Comment by joeb32** [comment]
thanks all for the help, its fixed and recording the temperatures now I've reinstalled Artisan with that latest version, not sure what the issue was 

Files below of successful Temp recording

[artisan.log](https://github.com/artisan-roaster-scope/artisan/files/7541698/artisa
[artisan_device.log](https://github.com/artisan-roaster-scope/artisan/files/7541699/artisan_device.log)
n.log)

**Comment by CarefreeBuzzBuzz** [comment]
You don’t see anything under the security tab in system preferences down at the bottom of that tab. You might have to hit the unlock button and put in your password and then approve it. If you don’t see this you might try installing the fidget striver again and see if it gives you any sort of warning.

Michael Herbert 
CarefreeBuzzBuzz
